app/services/force/event_subscriber_translate_service.rb

```diff
@@ -108,9 +113,15 @@ def parse_event(platform_event)
     end
 
     def extract_updated_values(changed_fields, event)
-      changed_fields.each_with_object({}) do |field, values|
+      updated_values = changed_fields.each_with_object({}) do |field, values|
```

Comment:
question: Do we still need to mutate `changed_fields`? If not we could use `map` instead:
```ruby
updated_values = changed_fields
                 .keys
                 .except('LastModifiedDate')
                 .map { |field| event.dig('payload', field) }
```